index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+
+</head>
+<body>
+    <div id="top_bar"> 
+        <div id="top1">
+          <img  src="img/top_bar1.png" alt="" >
+        </div >
+       
+            <div id="top2">
+                <input type="text" name="search" placeholder="Search" id="search">
+            
+           
+                <button id="button" type="submit">
+                    <img id="icon" src="img/search-icon.png" alt="">
+                </button>
+                        
+                
+            </div>
+        <div id="top3">
+          <img  src="img/top_bar2.png" alt="" >
+       </div>
+    </div>
+    <div id="bottom_part">
```

Comment:
please use section tag instead of div in line 31, for better utilizing HTML5 semantic tags as section tag is more semantic than div, also remove the id for the div(id="bottom_part") as it is not used in CSS file .